includes/documents/class-wcpdf-bulk-document.php

```diff
@@ -121,6 +130,59 @@ public function get_filename( $context = 'download', $args = array() ) {
 		return $filename;
 	}
 
+	protected function add_filters( $filters ) {
+		foreach ( $filters as $filter ) {
+			$filter = $this->normalize_filter_args( $filter );
+			add_filter( $filter['hook_name'], $filter['callback'], $filter['priority'], $filter['accepted_args'] );
+		}
+	}
+
+	protected function remove_filters( $filters ) {
+		foreach ( $filters as $filter ) {
+			$filter = $this->normalize_filter_args( $filter );
+			remove_filter( $filter['hook_name'], $filter['callback'], $filter['priority'] );
+		}
+	}
+
+	protected function normalize_filter_args( $filter ) {
```

Comment:
this all following functions should be removed (see previous comment)

Reply:
Fixed with https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/264/commits/b0c8d0988e545fd362c184348cdab82752d47452